cast/java/ecj/src/test/java/com/ibm/wala/cast/java/test/ECJJava15IRTest.java

```diff
@@ -39,241 +41,39 @@ protected Iterable<Entrypoint> makeDefaultEntrypoints(IClassHierarchy cha) {
     return engine;
   }
 
-  @Test
-  public void testVarargsOverriding()
+  static Stream<String> java15IRTestNames() {
+    return Stream.of(
+        "VarargsOverriding",
+        "EnumSwitch",
+        "OverridesOnePointFour",
+        "VarargsCovariant",
+        "SimpleEnums",
+        "GenericSuperSink",
+        "AnonGeneNullarySimple",
+        "AnonymousGenerics",
+        "NotSoSimpleEnums",
+        "Varargs",
+        "GenericArrays",
+        "Annotations",
+        "Cocovariant",
+        "ExplicitBoxingTest",
+        "Wildcards",
+        "BasicsGenerics",
+        "CustomGenericsAndFields",
+        "TypeInferencePrimAndStringOp",
+        "SimpleEnums2",
+        "GenericMemberClasses",
+        "MoreOverriddenGenerics");
```

Comment:
If the order of these items does not matter, then please sort them alphabetically. That will make this list easier to scan when adding new items, easier to avoid adding duplicates, etc..